### PR TITLE
CHAT-506 minichat popup: You have been kicked from this room on spaces

### PR DESCRIPTION
**Pull request by nttuyen**
Use SpaceID instead of prettyName when request the detail of space room.

### CODE REVIEW

File: application/src/main/webapp/js/notif.js [hunk 1]
```diff
@@ -679,7 +681,13 @@ ChatNotification.prototype.attachChatButtonBelowLeftNavigationSpaceName = functi
   var $btnChat = jqchat(".chat-button", $breadcumbEntry);
   var spaceName = this.shortSpaceName;
   if ($breadcumbEntry.length > 0 && $btnChat.length === 0 && spaceName !== "") {
-    var strChatLink = "<a onclick='javascript:showMiniChatPopup(\"" + spaceName + "\", \"space-name\");' class='chat-button actionIcon' href='javascript:void();'><span class='uiIconChatChat uiIconChatLightGray'></span><span class='chat-label-status'>&nbsp;Chat</span></a>";
+    var spaceId = this.spaceId;
+    var strChatLink;
+    if (spaceId !== '') {
+      strChatLink = "<a onclick='javascript:showMiniChatPopup(\"" + spaceId + "\", \"space-id\");' class='chat-button actionIcon' href='javascript:void();'><span class='uiIconChatChat uiIconChatLightGray'></span><span class='chat-label-status'>&nbsp;Chat</span></a>";
+    } else {
```
Comment (thomasdelhomenie) [inline application/src/main/webapp/js/notif.js]: Is there cases where we still use the space short name ?

Comment (nttuyen) [inline application/src/main/webapp/js/notif.js]: No, there isn't, we could always get the spaceId when in space context. I will remove this.

Comment (nttuyen) [inline application/src/main/webapp/js/notif.js]: PR updated.

File: application/src/main/java/org/exoplatform/chat/portlet/notification/NotificationApplication.java [hunk 2]
```diff
@@ -111,6 +111,7 @@ public NotificationApplication(OrganizationService organizationService, SpaceSer
     ResourceBundle bundle= applicationContext.resolveBundle(locale) ;
     String messages = bundleService_.getBundle("chatBundleData", bundle, locale);
     String shortSpaceName = getCurrentShortSpaceName();
```
Comment (thomasdelhomenie) [inline application/src/main/java/org/exoplatform/chat/portlet/notification/NotificationApplication.java]: Is shortSpaceName still used ?

**Pull request merged**